packages/action-menu/stories/action-menu.stories.ts

```diff
@@ -112,6 +114,24 @@ export default {
                 type: 'boolean',
             },
         },
+        staticValue: {
+            name: 'static',
+            type: { name: 'string', required: false },
+            description: 'The visual static variant to apply to the button.',
+            table: {
+                type: { summary: 'string' },
+                defaultValue: { summary: 'none' },
+            },
+            control: {
+                type: 'select',
+                labels: {
+                    white: 'white',
+                    black: 'black',
+                    none: undefined,
+                },
+                options: ['white', 'black', 'none'],
```

Comment:
I think `option` overrides `labels` in this case. I would expect choosing `none` not to publish into the DOM: 
<img width="239" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/1156657/8996f763-e56c-42c9-9706-b7f4a1702f5e">

Reply:
Right so the value of the selection will take from options and also if we add undefined in options it will override the labels. So I think there are two options we have:
1) We can add 'none' as a static variant and check for 'none' value in the update method and make it as empty. 
```
if (changedProperties.has('static') && this.static === 'none') {
            this.static ='';
}
```
2) add undefined as a variant like we have for other components in the code and proceed.

Reply:
I don't think we should be weighing down runtime code with work arounds for Storybook.

I got the way the `labels` field works quite wrong... 😞 

I think we should interpret `"none"` as `undefined` in the story, something like this:
```
        <sp-action-menu
            label=${ariaLabel}
            ?disabled=${disabled}
            ?open=${open}
            ?quiet=${quiet}
            static=${ifDefined(staticValue === 'none' ? undefined : staticValue)}
            size=${size}
            @change="${changeHandler}"
            .selects=${selects ? selects : undefined}
            value=${selected ? 'Select Inverse' : ''}
        >
```